nestkernel/event.h

```diff
@@ -952,6 +952,12 @@ class GapJunctionEvent : public SecondaryEvent
     supported_syn_ids_.push_back( synid );
   }
 
+  static void
+  set_coeff_length( const size_t coeff_length )
+  {
+    coeff_length_ = coeff_length;
```

Comment:
From the context it seems certain that this will never be called from a thread-parallel context, but if that should happen, we would be in trouble. Can we add an assertion here that we are running single-threaded, or explicitly mark this as critical section?